### PR TITLE
Rewrite Julia Basics

### PR DESCRIPTION
**Pull request by rikhuijzer**
I combined some time here and there to read through the section.

### CODE REVIEW

File: contents/julia_basics.md [hunk 1]
```diff
@@ -1037,196 +1051,175 @@ We also have some **syntax aliases** for the most common elements in array const
...
     ```jl
-    sco(
-    """
-    my_vector_zeros = zeros(10)
-    """
-    )
+    s = """
+        my_vector_zeros = zeros(10)
+        """
+    sco(s)
```
Comment (rikhuijzer) [inline contents/julia_basics.md:1057]:
```suggestion
    sco("my_vector_zeros = zeros(10)")
```
Would have been better, but I forgot to do it here. Maybe next time. Later in the PR, I did go for the shorter form.

**Review by storopoli — COMMENTED** [review]
Ok, so the idea is to remove the initial bullet points to a more narrative approach?

File: contents/julia_basics.md (hunk 1 above)
Comment (storopoli) [reply contents/julia_basics.md:1057]: There are several like this in the file. Doesn't make sense to change only one...

**Comment by rikhuijzer** [comment]
> Ok, so the idea is to remove the initial bullet points to a more narrative approach?

No. I didn't remove any pullet point. I only did did a rewrite of some small sentences, added some sentences in places where I got confused and I did rewrite
```
sco("""
    foo
    """
)
```
to either
```
s = """
     foo
     """
sco(s)
```
or
```
sco("foo")
```

Let me know if I should revert anything

**Review by storopoli — APPROVED** [review]
Ok, thanks! The improvements makes the text better and with the "face" of the three of us. Instead of just one author.

**Comment by rikhuijzer** [comment]
Thank you for your review, Jose!

**Pull request merged**